gandalf/build.gradle

```diff
@@ -63,40 +64,12 @@ gnag {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
 
-    compile 'com.android.support:appcompat-v7:25.3.1'
-    compile 'com.squareup.okhttp3:okhttp:3.6.0'
-    compile 'com.google.code.gson:gson:2.8.0'
+    implementation 'androidx.appcompat:appcompat:1.0.2'
+    implementation 'com.squareup.okhttp3:okhttp:4.0.1'
+    implementation 'com.google.code.gson:gson:2.8.5'
 
-    testCompile 'junit:junit:4.12'
-    testCompile 'org.mockito:mockito-core:2.7.13'
-}
-
-ext {
-    bintrayRepo = 'maven'
-    bintrayName = 'gandalf'
-
-    publishedGroupId = 'com.btkelly'
-
-    artifact = 'gandalf'
-
-    libraryName = 'Gandalf'
-    libraryDescription = 'Easily notify a user with a simple alert, inform them of an optional update, and in dire situations block the user from accessing older versions of the application completely.'
-    libraryVersion = versionString
-    libraryIssueTrackerUrl = 'https://github.com/btkelly/gandalf/issues'
-    libraryLabels = ['android', 'update']
-
-    licenseName = 'Apache License Version 2.0'
-    licenseUrl = 'http://www.apache.org/licenses/LICENSE-2.0.html'
-
-    gitUrl = 'https://github.com/btkelly/gandalf'
-    siteUrl = 'http://btkelly.github.io/gandalf'
-
-    developerId = 'btkelly'
-    developerName = 'Bryan Kelly'
-    developerEmail = 'bobbake4@gmail.com'
-}
-
-apply from: 'https://raw.githubusercontent.com/btkelly/JCenter/f988f0e106be89c876e038894b9892f9d1d0b643/bintrayv1.gradle'
-apply from: 'https://raw.githubusercontent.com/btkelly/JCenter/master/installv1.gradle'
+    testImplementation 'junit:junit:4.12'
+    testImplementation 'org.mockito:mockito-core:2.7.13'
```

Comment:
Can we update this to `3.0.0` to fix support for newer JDK versions?

Reply:
Done in https://github.com/btkelly/gandalf/pull/87/commits/6acaec4225df7a82e5fff584f4280d46d30d5eb1